src/components/Inventory/NurseryDropdown.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useState } from 'react';
+import { AccessionPostRequestBody } from 'src/api/accessions2/accession';
+import strings from 'src/strings';
+import { ServerOrganization } from 'src/types/Organization';
+import { SelectT } from '@terraware/web-components';
+import { Facility } from 'src/api/types/facilities';
+import { getAllNurseries } from 'src/utils/organization';
+
+interface NurseryDropdownProps<T extends AccessionPostRequestBody> {
+  organization: ServerOrganization;
+  record: T;
+  setRecord: React.Dispatch<React.SetStateAction<T>>;
+  disabled?: boolean;
+  validate?: boolean;
+}
+
+export default function NurseryDropdown<T extends AccessionPostRequestBody>(
+  props: NurseryDropdownProps<T>
+): JSX.Element {
+  const { organization, setRecord, validate, record } = props;
+  const [selectedValue, setSelectedValue] = useState<Facility>();
+
+  const onChangeHandler = (value: Facility) => {
+    setSelectedValue(value);
+    if (value?.id) {
+      setRecord((previousRecord: T): T => {
+        return {
+          ...previousRecord,
+          facilityId: value.id,
+        };
+      });
+    }
+  };
+
+  const toT = (option: string) => {
+    return { name: option } as Facility;
+  };
+
+  const getNurseries = () => {
+    const nurseries: Facility[] = [];
+    getAllNurseries(organization).forEach((nursery) => {
+      if (nursery !== undefined) {
+        nurseries.push(nursery);
+      }
+    });
+    return nurseries;
+  };
+
+  return (
+    <SelectT<Facility>
```

Comment:
i think we added a Dropdown component that captures this
https://github.com/terraware/web-components/blob/main/src/components/Dropdown.tsx#L21

Reply:
Maybe seedbank selector can also use the Dropdown (it was developer before that?).
Can we map the nurseries to { label: facility.name, value: facility.id } ?
<Dropdown
  id='facilityId'
  label: 'Nurseries'
  items={nurseries.map(nursery=> ({ label: nursery.name, value: nursery.id }))}
  onChange={onChangeHandler}
/>

where 
onChangeHandler = (facilityId) => {
  setRecord(....
     facilityId: Number(facilityId),
  ) 
}